ajax/libs/react-mdl/package.json

```diff
@@ -0,0 +1,62 @@
+{
+  "name": "react-mdl",
+  "filename": "ReactMDL.min.js",
+  "version": "1.10.3",
+  "description": "React Components for Material Design Lite",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/react-mdl/react-mdl"
+  },
+  "homepage": "https://react-mdl.github.io/react-mdl/",
```

Comment:
This link show site not found.
The like should be: https://tleunen.github.io/react-mdl/?
What do you think?
Thanks.